website/static/css/team.css

```diff
@@ -0,0 +1,137 @@
+.page_header {
+  text-align: center;
+  padding-bottom: 9px;
+  margin: 40px 0 20px;
+  border-bottom: 1px solid #eeeeee;
+}
+
+.page_header>h1 {
+  font-size: 36px;
+  margin-top: 20px;
+  margin-bottom: 10px;
+}
+
+.page_header>p {
+  font-size: 17px;
+  font-weight: 300;
+  line-height: 1.4;
+  margin: 5px;
+}
+
+.page_header>p:hover {
+  text-decoration: underline;
+}
+
+.member_orgs {
+  display: flex;
+}
+
+.org_image > img {
+  height: 22px;
+  width: 22px;
+  border-radius: 5px;
+  margin-right: 2px;
+}
+
+.member_avatar > img {
+  height: 80px;
+  width: 80px; 
+  border-radius: 5px;
+  float: left;
+  margin-right: 20px;
+}
+
+
+.anchor_text {
+  color: #23527c;
+  font-size: 14px;
+}
+
+.anchor_text:hover {
+  text-decoration: underline;
+}
+
+.team_member {
+  box-sizing: border-box;
+  display: flex;
+  height: 100px;
+  margin-bottom: 10px;
+}
+
+.member_info {
+  display: block;
+  margin-right: 20px;
+  line-height: 1.7;
+}
+
+.member_block {
+  display: flex;
+  flex-wrap: wrap;
+  align-content: space-between;
+}
+
+/* 
+  ##Device = Desktops
+  ##Screen = 1281px to higher resolution desktops
+*/
+
+@media (min-width: 1281px) {
+  .team_member {
+    width: 25%;

```

Comment:
Does flexbox itself not cover this kind of thing already?

Reply:
Yes. You can achieve the same thing using flex-box as well. I have used media queries to set specific styles based on the screen width. Flexbox helps in layout settings without knowing explicit width, height values. Also, we can change flex layouts within media queries as well. Shall I do do the same using flexbox ?

Reply:
To be a bit more clear, flex box has more to do with how space is being used and media queries tells how content is being viewed.